### PR TITLE
feat(ali-oss): add policy operations and calculatePostSignaturepolicy

### PR DESCRIPTION
**Pull request by CodyTseng**
Add policy operations.
Add calculatePostSignature.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/ali-sdk/ali-oss#putBucketPolicyname-policy-options>><<https://github.com/ali-sdk/ali-oss#calculatePostSignaturepolicy>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by elibarzilay** [comment]
As you wait for reviews, please consider adding tests (see typescript-bot's comment).

File: types/ali-oss/index.d.ts [hunk 1]
```diff
@@ -915,6 +937,35 @@ declare class OSS {
...
+
+    /**
+     * Obtains the policy for a bucket.
+     */
+    getBucketPolicy(name: string, options?: OSS.RequestOptions): Promise<OSS.GetBucketPolicyResult>;
+
+    /**
+     * Deletes the policy added for a bucket.
+     */
+    deleteBuckyPolicy(
```
Comment (weswigham) [inline types/ali-oss/index.d.ts:961]: Typo? `Bucky`

**Review by weswigham — CHANGES_REQUESTED** [review]

File: types/ali-oss/index.d.ts (hunk 1 above)
Comment (CodyTseng) [reply types/ali-oss/index.d.ts:961]: Thank you for your review! I will fix the typo and test it as soon as possible.

**Review by weswigham — APPROVED** [review]

**Comment by CodyTseng** [comment]
Ready to merge

**Pull request merged**